gridstatus/isone_api/isone_api.py

```diff
@@ -65,7 +66,10 @@ def __init__(
         self.initial_delay = min(sleep_seconds, 60.0)
         self.max_retries = min(max(0, max_retries), 10)
 
-    # TODO(kladar) abstract this out to a base class since it is shared with ERCOT API logic
+    def parse_problematic_datetime(self, date_string: str) -> pd.Timestamp:
+        dt = datetime.strptime(date_string, "%Y-%m-%dT%H:%M:%S.%f%z")
```

Comment:
Can we just pass this in as `format` to `pd.to_datetime`? Or, is the format for some time strings different?

Reply:
Yeah, I tried that (and a few other approaches that are similar) but the issue was with the way the pandas `dt` accessor handled dates in the object, all throwing this common `dt` error. So had to come up with a way to avoid the pandas accessor methods and fall back to `datetime` https://stackoverflow.com/questions/68734962/pandas-dataframe-column-can-only-use-dt-accessor-with-datetimelike-values

So my current hypothesis it that it's less of a string format issue and more of a duck-typing failure on the pandas end. Only happens on DST conversion days, and happens on every one of them. 

But also points out that the return type is wrong here, just updated that.

Reply:
Got it, thanks for the explanation.